www/index.php

```diff
@@ -377,6 +377,15 @@
                                         <input id="time" type="text" class="text short" name="time" value=""> seconds
                                     </li>
                                     <li>
+									<li>
+                                        <label for="time">

```

Comment:
If Pat is okay with having it in the UI, I don't think you need a corresponding boolean in the .ini file. Right? The fact that someone specified a non-zero timeout value in the UI should be enough for the code to know that a per-test timeout is enabled.